shared/src/packages/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -1,6 +1,7 @@
 const UserValidationMessage = {
   EMAIL_REQUIRED: 'Email is required',
   EMAIL_WRONG: 'Email is wrong',
+  NAME_REQUIRED: 'Name is required',
```

Comment:
```suggestion
  NAME_REQUIRED: 'Name is required.',
```
Please, put dot at the end of each message here